JSDemos/Demos/Common/DialogsAndNotificationsOverview/jQuery/index.js

```diff
@@ -1,6 +1,11 @@
 $(() => {
   let currentHouse;
 
+  DevExpress.setTemplateEngine({
+    compile: (element) => DevExpress.utils.dom.extractTemplateMarkup(element),
```

Comment:
Don't use private methods in demos. Use JQuery to extract markup - $(element).html()